### PR TITLE
Ne pas afficher comme résultat les presta injectées

### PR DESCRIPTION
**Pull request by fpagnoux**
Connected to sgmap/mes-aides-ui#118

C'est mal fait pour le moment (mapping calculé deux fois), mais pour servir de base à discussion demain :)

### CODE REVIEW

**Comment by fpagnoux** [comment]
@MattiSG, I'm done with this one too ;)

File: lib/simulation/openfisca/index.js [hunk 1]
```diff
@@ -40,10 +40,15 @@ function calculate(situation, debug, callback) {
 }
 
 function simulate(situation, callback) {
+    var injectedRessources = _.unique(_.flatten(_.map(situation.individus, function(individu) {
+        return _.uniq(_.map(individu.ressources, function(ressource) {
+            return ressource.type;
```
Comment (MattiSG) [inline lib/simulation/openfisca/index.js]: If you're simply accessing a prop, use the `String` version of the iteratee  :)
(in which case you can also improve readability by using the `collect` synonym of `map`)

File: lib/simulation/openfisca/index.js [hunk 2]
```diff
@@ -40,10 +40,15 @@ function calculate(situation, debug, callback) {
 }
 
 function simulate(situation, callback) {
+    var injectedRessources = _.unique(_.flatten(_.map(situation.individus, function(individu) {
+        return _.uniq(_.map(individu.ressources, function(ressource) {
```
Comment (MattiSG) [inline lib/simulation/openfisca/index.js]: No need to `unique` twice. Also, be consistent between `uniq` and `unique` synonyms.

**Comment by MattiSG** [comment]
GTM

**Comment by fpagnoux** [comment]
Done ;)

**Comment by MattiSG** [comment]
GTM

**Pull request merged**